GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListViewModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DogBreedsListViewModel.swift
+//  GeorgianMemenator
+//
+//  Created by Qeti Avanashvili on 08.01.23.
+//  Copyright © 2023 Shalva Avanashvili. All rights reserved.
+//
+
+import SwiftUI
+import Foundation
+
+class DogBreedsListViewModel: ObservableObject {
+    
```

Comment:
Please format this file. No new lines and unneeded spaces.

Reply:
I have corrected everything, hope there won't be any problem 🙏🙏